src/mps/projmposum.jl

```diff
@@ -23,115 +26,63 @@ sum Σⱼ on the left):
      o--o--o-      -o--o--o--o--o--o |psi>
 ```
 """
-mutable struct ProjMPOSum
-  pm::Vector{ProjMPO}
-end
 
-copy(P::ProjMPOSum) = ProjMPOSum(copy.(P.pm))
+copy(P::ProjSum{T}) where {T} = ProjSum{T}(copy.(P.pm))
```

Comment:
It should work without the `where {T}`:
```julia
copy(P::ProjSum) = ProjSum(copy.(P.pm))
```
Julia should be able to deduce the type automatically through default constructors that it makes.